packages/wp-now/README.md

```diff
@@ -162,6 +162,19 @@ Some similarities between `wp-env` and `wp-now` to be aware of:
 -   deployments are not possible with neither `wp-env`, nor `wp-now`;
 -   possible to switch easily the PHP version
 
+## Publishing to npm
+
+The `wp-now` package is part of a larger monorepo, sharing its space with other sibiling packages. To publish the `wp-now` package to npm, you must first understand the automated release process facilitated by lerna. This process includes automatically incrementing the version number, creating a new tag, and publishing all modified packages to npm simultaneously. Notably, all published packages share the same version number.
+
+Each package identifies a distinct organization in its `package.json` file. To publish the `wp-now` package, you need access to the npm organizations `@wp-playground`, `@php-wasm`, and `@wp-now`.
```

Comment:
Who should publish releases?

Reply:
This is related to governance, I'll kick off that project soon